scalameta/tests/jvm/src/test/resources/semanticdb.expect

```diff
@@ -88,6 +84,16 @@ Synthetics:
   [13..16): Int => _root_.scala.reflect.ClassTag.Int.
 
 
+scalameta/semanticdb-integration/src/main/scala/example/Example.scala
```

Comment:
We are not merging documents as that would require reading them from disk first, instead we just append documents which means they will get output as a separate document when calling syntax on the database.